Documentation/README.md

```diff
@@ -1,64 +1,181 @@
+# DotNet Issue Labeler
+
+This document describes how to set up a new issue labeler app (for a new GitHub repo), as well as how to update the training data for an existing labeler app and repo.
+
+Note: Several parts of this document have steps that require particular membership and permissions for various GitHub repos, Azure subscriptions and resources, and so forth. You may have to request permissions or send PRs to make certain changes.
+
+## Add support for a new GitHub repo
+
+If your repo is not yet set up for issue labeling, there are two options:
+
+1. Use an existing labeler app that supports the same GitHub org and add your repo's data to it
+1. Create a new labeler app and add your repo's data to it
+
+Please note that a single labeler app can support only one org, so if your org isn't supported yet, you must create a new labeler app instance.
+
+If your are adding your repo to an existing labeler, see the [Web App Service List](#web-app-service-list) for a list of existing apps.
+
+Follow these steps to create a new labeler app:
+
+1. To create a new labeler, you must have access to the `GitHubIssueLabeller` resource group in the `DDFun IaaS Dev Shared Public` Azure Subscription. Contact the DevDiv Azure Ops email alias to request access.
```

Comment:
Is there an easy fix to the spelling of `GitHubIssueLabeller`? If not, what's the long-term solution here?

Reply:
Unfortunately it does not seem possible to rename an Azure Resource Group at this time, so it's stuck. We could create a new group and move everything to it, but that would likely break various things. I think the long-term solution is that we learn as best we can to live with this mistake 😄